go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -71,9 +71,20 @@ type EmergencyReparentOptions struct {
 
 // counters for Emergency Reparent Shard
 var (
-	ersCounter        = stats.NewGauge("ers_counter", "Number of times Emergency Reparent Shard has been run")
-	ersSuccessCounter = stats.NewGauge("ers_success_counter", "Number of times Emergency Reparent Shard has succeeded")
-	ersFailureCounter = stats.NewGauge("ers_failure_counter", "Number of times Emergency Reparent Shard has failed")
+	// TODO(timvaillancourt): remove legacyERS* gauges in v19+.
+	legacyERSCounter        = stats.NewGauge("ers_counter", "Number of times Emergency Reparent Shard has been run")
+	legacyERSSuccessCounter = stats.NewGauge("ers_success_counter", "Number of times Emergency Reparent Shard has succeeded")
+	legacyERSFailureCounter = stats.NewGauge("ers_failure_counter", "Number of times Emergency Reparent Shard has failed")
+
+	ersCounter = stats.NewCountersWithMultiLabels("emergency_reparent_shards", "Number of times Emergency Reparent Shard has been run",
```

Comment:
I wonder if this can be collapsed into one stat with an additional label called result.
So that we could have something like:
```
vtctld_emergency_reparent_counts{keyspace="commerce",shard="-80",result="success"} 1
vtctld_emergency_reparent_counts{keyspace="commerce",shard="-80",result="failure"} 0
vtctld_emergency_reparent_counts{keyspace="commerce",shard="80-",result="success"} 1
vtctld_emergency_reparent_counts{keyspace="commerce",shard="80-",result="failure"} 0
```
Notice that I'm proposing dropping `shards` from the metric names and replacing it with `counts`.
I have the same suggestion for `planned_reparent` stats too.